velox/exec/benchmarks/HashJoinListResultBenchmark.cpp

```diff
@@ -23,6 +23,9 @@
 #include <folly/init/Init.h>
 #include <iostream>
 
+DEFINE_bool(gtest_color, false, "");
```

Comment:
Remove these

Reply:
@yingsu00 Masha commented internally that we should be consistent (stick to `std`), so let's just put `randomEngine_(std::random_device{}())` here

Reply:
@Yuhta Thanks for the suggestion! Just updated the PR as suggested.